boards/arm/nrf52832_mdk/nrf52832_mdk.dts

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2018 Zelin <zelin@makerdiary.com>
+ * Copyright (c) 2018 makerdiary.com.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <nordic/nrf52832.dtsi>
```

Comment:
change to include <nordic/nrf52832_qfaa.dtsi>

Reply:
ok

Reply:
ok